store.js

```diff
@@ -0,0 +1,93 @@
+const { createStore } = require('./redux.min');
+
+const initialState = {
+  itemList: ['Apple', 'Bread', 'Carrot', 'Dumplings', 'Eggs', 'Fish', 'Garlic', 'Honey', 'Ice cream', 'Jam'],
+  selectedItem: null,
+}
+
+const actions = {
+  moveUp: 'moveUp',
+  moveDown: 'moveDown',
+  select: 'select',
+}
```

Comment:
экшены называем в верхнем регистре
```suggestion
const actions = {
  MOVE_UP: 'MOVE_UP',
  moveDown: 'moveDown',
  select: 'select',
}
```